web/src/lib/components/StallView.svelte

```diff
@@ -178,12 +187,95 @@
 
   <!-- <CampaignStats /> -->
 {:else}
-  <!-- always keep a 3:1 aspect ratio, see https://stackoverflow.com/a/12121309 -->
-  <div class="lg:w-2/3 mx-auto">
-    <div class="w-full inline-block relative after:pt-[33.33%] after:block after:content-[''] lg:mt-24 mt-12">
-          <div class="absolute top-0 bottom-0 left-0 right-0 rounded-md bg-center bg-no-repeat bg-cover" style="background-image: url({bannerUrl});" alt=""></div>
+ <!-- always keep a 3:1 aspect ratio, see https://stackoverflow.com/a/12121309 -->
+ <div class="mx-auto relative lg:mb-0 mb-36">
+  <div class="absolute inset-x-0 lg:bottom-20 m-auto left-0 right-0 lg:w-2/3 mx-auto z-40">
+    <div class="grid lg:grid-cols-2 gap-4">
+      <!-- COL1 -->
+      <div class="lg:flex space-x-8 items-center w-full">
+        <div class="grid lg:place-items-start place-items-center">
+          <div class="grid place-items-center my-4">
+            {#if owner}
+                <Avatar account={owner} size={AvatarSize.M}  />
+            {/if}
+          </div>
+          <h2 class="lg:text-5xl text-4xl font-bold">{title}</h2>
+          {#if description}
+              <div class="markdown-container leading-8 my-2 lg:text-left text-center p-1">
+                  <SvelteMarkdown source={description} />
+              </div>
+          {/if}
+          <div class="lg:flex items-center lg:space-x-2">
+            <!-- TELEGRAM -->
+            {#if telegramHref}
```

Comment:
I don't get this... you are adding telegram here, but it was already on this page. Does it mean we will have it twice now?